corgi/core/models.py

```diff
@@ -225,15 +228,16 @@ class Type(models.TextChoices):
     type = models.CharField(choices=Type.choices, max_length=20)
     name = models.TextField()  # Arbitrary identifier for a build
     source = models.TextField()  # Source code reference for build
+    completion_time = models.DateTimeField(null=True)  # meta_attr["completion_time"]
```

Comment:
I think this should be a non-nullable field. Each build must have a completed time, otherwise it's not completed yet and we don't want to collect it.

Reply:
that was my instinct as well, though there are software builds without completion time ;( ... unsure of what the right answer should be

Reply:
Got an example?

Reply:
hmmm looping through SoftwareBuild - all have completion time... weird ... maybe the err I was seeing was converting it into datatime ... one moment